server-api/server.js

```diff
@@ -36,6 +36,7 @@ server.get("/static/*", (req, res) => {
 
 // REST API
 server.get("/customers/coords", customerLocationsHandler.allCustomerLocations)
+server.get("/customers/:email", customers.getSpecificCustomer)
```

Comment:
Not sure we should use sensitive data in the URL  https://www.w3schools.com/tags/ref_httpmethods.asp